azext_iot/iothub/params.py

```diff
@@ -246,3 +248,73 @@ def load_iothub_arguments(self, _):
             options_list=["--content-type", "--ct"],
             help="MIME Type of file.",
         )
+
+    with self.argument_context("iot hub state") as context:
+        context.argument(
+            "state_file",
+            options_list=["--state-file", "-f"],
+            help="The path to the file where the state information will be stored."
+        )
+        context.argument(
+            "replace",
+            options_list=["--replace", "-r"],
+            help="If this flag is set, then the command will delete the current devices, configurations, and certificates "
+                 "of the destination hub."
+        )
+        context.argument(
+            "hub_aspects",
+            options_list=["--aspects"],
+            nargs="*",
+            arg_type=get_enum_type(HubAspects),
+            help="Hub Aspects (space seperated)."
```

Comment:
spelling seperated -> separated , also in other areas we have been using a dash

![image](https://user-images.githubusercontent.com/28658112/197639381-5e915b4a-a6cc-4f54-ab2c-0698a225f013.png)

Reply:
ok makes sense - if a user types in --aspects, we expect them to type in the specific aspect(s) and if they ignore the param, then we do all